packages/amplify_auth_cognito/example/tool/provision_integration_test_resources.sh

```diff
@@ -2,6 +2,9 @@
 set -e
 IFS='|'
 
+# In development, AWS_PROFILE should be set. In CI, it's not.
```

Comment:
nit: please remove last bit about CI or just remove this comment as we will not run this in CI. (my bad as I suggested this initially)